data/brands/amenity/restaurant.json

```diff
@@ -2786,7 +2786,7 @@
       "displayName": "Ponderosa Steakhouse",
       "id": "ponderosasteakhouse-e617a8",
       "locationSet": {
-        "include": ["ae", "eg", "qa", "us"]
+        "include": ["ae", "eg", "qa", "tw", "us"]
```

Comment:
this will cause you find 2 item when you tag, one named ```Ponderosa Steakhouse```, and another named ```龐德羅莎``` (at L2805)